### DISCUSSION TITLE
How to disable auto insert domain name to links

### DISCUSSION THREAD

**Discussion by monkeydev001** [JiHong88/SunEditor#1321] [Q&A]
I use Sun Editor to edit email templates and generate links for another website using primitives like {$site_url}. 
However, the Link plugin automatically corrects and adds the current website's URL to the href attribute.   
How can I disable this feature?

This is an example of a generated link:
`<a  href="https:\\someurl.com\{$site_url}\restore-access\{$hash}"> 
{$site_url}\restore-access\{$hash}
</a>
`

**Comment by JiHong88** [comment]
You can used this option. `linkNoPrefix: true`
